app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LikesController < ApplicationController
+  before_action :find_specific_post, only: [:like]
+
+  def new
+    @like = Like.new
+  end
+
+  def create
+    return if liked?
+
+    redirect_to "/users/#{params[:user_id]}"
+
+    Like.create(like_params)
+    redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id]), notice: 'Like Added!'
```

Comment:
- [ ] The requirements expect us to do this for all create actions in all controllers 👇🏼.

  > - Add flash messages in the create actions in all your controllers.
  
  Just like you've done in the post and comment controllers, please add flash messages for the successful and the unsuccessful paths.

Reply:
This issue is well resolved! ✔️